qiita_pet/handlers/analysis_handlers/base_handlers.py

```diff
@@ -0,0 +1,101 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from tornado.web import authenticated
+
+from qiita_core.util import execute_as_transaction
+from qiita_core.qiita_settings import qiita_config
+from qiita_pet.handlers.base_handlers import BaseHandler
+from qiita_pet.handlers.analysis_handlers import check_analysis_access
+from qiita_pet.handlers.util import to_int
+from qiita_db.analysis import Analysis
+
+
+class CreateAnalysisHandler(BaseHandler):
+    @authenticated
+    @execute_as_transaction
+    def post(self):
+        name = self.get_argument('name')
+        desc = self.get_argument('description')
```

Comment:
No it's not required (same behavior as before). If the user doesn't introduce anything, the description will be blank (empty string).